client/packages/system/src/Location/api/api.ts

```diff
@@ -5,13 +5,15 @@ import {
   InsertLocationInput,
   UpdateLocationInput,
   DeleteLocationInput,
+  FilterByWithBoolean,
 } from '@openmsupply-client/common';
 import { Sdk, LocationRowFragment } from './operations.generated';
 
 export type ListParams = {
   sortBy: SortBy<LocationRowFragment>;
   first?: number;
   offset?: number;
+  filterBy: FilterByWithBoolean | null;
```

Comment:
Suggestion from @clemens-msupply who picked me up on this in another PR 😉 
```suggestion
  filterBy: LocationFilterInput | null;
```